contribute_to_pim/tests/establishing_decorator_pattern.rst

```diff
@@ -86,7 +86,7 @@ Navigate And Change The Current Page
     }
 
 Selenium will know which URL to go to thanks to `the protected $path attribute
-<https://github.com/akeneo/pim-community-dev/blob/master/features/Context/Page/Role/Edit.php#L19>`_
+<https://github.com/akeneo/pim-community-dev/blob/master/tests/legacy/features/Context/Page/Role/Edit.php#L19>`_
```

Comment:
If you keep 'master' links, you should add an advice on the process of the branches creation (see with @SamirBoulil  if there is a doc for that) to replace the `master` by `3.0` when the branch is created, no ?
I did a doc on Jira for the Dockerfile update, I don't know if it's on the samir's doc.

Reply:
yeah ok
I'll merge the PR, and see with @SamirBoulil what I should do about that